### DISCUSSION TITLE
RWD theme in LUMA style

### DISCUSSION THREAD

**Discussion by Danieleeffe1** [OpenMage/magento-lts#2124] [Ideas]
### Change theme style
By replacing 2 template files and the css file you get the look very close to the magento2 LUMA theme.
change file:
app/design/frontend/rwd/template/page/html/header.phtml
skin/frontend/rwd/css/style.css
skin/frontend/rwd/js/app.js

### Benefits
A more updated aspect, I use mobile menu always present

### Additional information
I modified the template files to make the menus and cart more suitable for mobile devices

You could add a LUMARWD theme in Designe / frontend / rwd and in skin / frontend / rwd

file:
[styles (1).zip](https://github.com/OpenMage/magento-lts/files/6350180/styles.1.zip)
[app.zip](https://github.com/OpenMage/magento-lts/files/6350182/app.zip)
[header.phtml.zip](https://github.com/OpenMage/magento-lts/files/6350183/header.phtml.zip)

live exemple :
https://test.monsterbox.it

**Comment by fballiano** [comment]
I'd really like to have a new (default) frontend theme but at the same time I think this would require an RFC process, as it would probably break previous code.

If it's not the default theme then I woudln't add it to the main repo, as it's so easy to install it from an external resource (did you publish it somewhere?). If you published it then we could promote it on https://openmage.readthedocs.io/en/latest/modules/themes.html

**Comment by fballiano** [comment]
Also, did you use code from M2? from the files you uploaded it seems so and I don't think the licensed are compatible (AFL-OSL) but anyways it's better to keep M2 copyrighted materials out of this repo.

**Comment by Danieleeffe1** [comment]
No, Is rdw custom file

in detail: 
in style.css modified color padding parameters and responsive rules
in app.js modified the parameters that compress the layered
in header.phtml added a div to implement the full width header


Il mar 27 apr 2021, 11:51 Fabrizio Balliano ***@***.***> ha
scritto:

**Comment by fballiano** [comment]
ah ok ok, sorry! I never used rwd and I didn't remember it was part of M1 :-(

I'd anyway prefer to keep it in a separate repo which, should then be installed under design/frontend/rwd/luma, what do you think about that?

**Comment by Danieleeffe1** [comment]
I agree.  I think it is the best solution.
